### ISSUE TITLE
Support exchanger GMO and Liquid

### ISSUE THREAD

**Issue by leonwang908** [bmoscon/cryptofeed#800]
Thanks for your great work for this library,
Is that possible to support below 2 exchangers in future?

GMO
https://api.coin.z.com/docs/en/#end-point

Liquid
https://developers.liquid.com/

**Comment by bmoscon** [comment]
yes those can both be added. I can probably do that by end of month.

**Comment by leonwang893** [comment]
That's great! Thanks a lot, your my hero!

Btw,
The GMO one's trade data is a little bit trick and it not mentioned in the document that,
in order to get correct trade data.
 "option": "TAKER_ONLY" is needed when sending message like below.

websocket.enableTrace(True)
ws = websocket.WebSocketApp('wss://api.coin.z.com/ws/public/v1')

def on_open(self):
    message = {
        "command": "subscribe",
        "channel": "trades",
        "symbol": "BTC",
        "option": "TAKER_ONLY"
    }
    ws.send(json.dumps(message))

https://api.coin.z.com/docs/en/#ws-trades

**Comment by leonwang893** [comment]
Hello,

May I know if there are anything that I can help?
